### ISSUE TITLE
The question about the loss function

### ISSUE THREAD

**Issue by xiutangzju** [guitaowufeng/TPANN#3]
In the paper of DANN, "While the parameters of the classifiers are optimized in order to minimize their error on the training set, the parameters of the underlying deep feature mapping are optimized in order to minimize the loss of the label classifier and to maximize the loss of the domain classifier. "
Why the loss functions are minimized? The loss function of the domain classifier should not be maximized?

**Comment by guitaowufeng** [comment]
Sorry for the late reply. Both the loss function should be minimized. We maximize the loss through the GRL layer on the feature extraction layer.
